save-backend/src/main/kotlin/com/saveourtool/save/backend/controllers/TestExecutionController.kt

```diff
@@ -53,29 +54,30 @@ class TestExecutionController(private val testExecutionService: TestExecutionSer
      * @param executionId an ID of Execution to group TestExecutions
      * @param page a zero-based index of page of data
      * @param size size of page
-     * @param status
-     * @param testSuite
+     * @param filters
      * @param authentication
      * @param checkDebugInfo if true, response will contain information about whether debug info data is available for this test execution
      * @return a list of [TestExecutionDto]s
      */
-    @GetMapping(path = ["/api/$v1/testExecutions"])
+    @PostMapping("/api/$v1/filteredTestExecutions")
     @RequiresAuthorizationSourceHeader
     @Suppress("LongParameterList", "TOO_MANY_PARAMETERS", "TYPE_ALIAS")
     fun getTestExecutions(
         @RequestParam executionId: Long,
         @RequestParam page: Int,
         @RequestParam size: Int,
-        @RequestParam(required = false) status: TestResultStatus?,
-        @RequestParam(required = false) testSuite: String?,
+        @RequestBody(required = false) filters: TestExecutionFilters = TestExecutionFilters.empty,
```

Comment:
Isn't it better to use `@RequestBody(required = false, defaultValue = TestExecutionFilters.empty)` if it's possible?

Reply:
oh sure, my bad